imagick/memory.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 Herbert G. Fischer. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package imagick
+
+/*
+#cgo !no_pkgconfig pkg-config: MagickWand MagickCore
+#include <wand/MagickWand.h>
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+// Relinquishes memory resources returned by such methods as MagickIdentifyImage(), MagickGetException(), etc
+func RelinquishMemory(ptr unsafe.Pointer) {
```

Comment:
Should this function, alone with `RelinquishMemoryCStringArray` be package private? Is there any reason we can think of for users of the package to need to free unsafe pointers? My guess (and maybe I am wrong) is that we are trying to hide the details of the memory management.